pkgs/development/python-modules/sphinx/default.nix

```diff
@@ -25,7 +25,7 @@
 }:
 
 buildPythonPackage rec {
-  pname = "sphinx";
+  pname = "Sphinx";
```

Comment:
@Synthetica9 Was there a particular reason for the change? It causes sphinx to fail to build so I reverted it.

Reply:
See #50269